job/job_test.go

```diff
@@ -173,3 +175,46 @@ func TestSetupFails(t *testing.T) {
 	// should still be able to log, see #_incident-130
 	j.With("happyThoughts", true).Error(err)
 }
+
+func TestDiscoverActiveWorkflowForMacFound(t *testing.T) {
```

Comment:
Can you write an assertion that checks the actual return? For example, can you assert that `status` is `true`?

Reply:
done